gitlabform/gitlab/members.py

```diff
@@ -9,12 +9,6 @@ def get_project_members(self, project_and_group_name, all=False):
 
         return self._make_requests_to_api(url_template, project_and_group_name)
 
-    def get_shared_with_groups(self, project_and_group_name):
-        # a dict with groups that this project has been shared with
-        return self._make_requests_to_api("projects/%s", project_and_group_name)[
-            "shared_with_groups"
-        ]
```

Comment:
Removed because this attribute can be accessed directly from a python-gitlab `Project` instance.